### PR TITLE
Standardize logging

### PR DESCRIPTION
**Pull request by labrys**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/metadata/mede8er.py [hunk 1]
```diff
@@ -335,24 +327,28 @@ def write_show_file(self, show_obj):
 
         try:
             if not os.path.isdir(nfo_file_dir):
-                logger.log(u'Metadata directory did not exist, creating it at {path}'.format
-                           (path=nfo_file_dir), logger.DEBUG)
+                log.debug(
+                    u'Metadata directory did not exist, creating it at {location}',
```
Comment (p0psicles) [inline medusa/metadata/mede8er.py]: Location vs path

File: medusa/metadata/mede8er.py [hunk 2]
```diff
@@ -335,24 +327,28 @@ def write_show_file(self, show_obj):
...
+                    {'path': nfo_file_dir}
+                )
                 os.makedirs(nfo_file_dir)
                 helpers.chmod_as_parent(nfo_file_dir)
 
-            logger.log(u'Writing show nfo file to {path}'.format
-                       (path=nfo_file_path), logger.DEBUG)
+            log.debug(
+                u'Writing show nfo file to {location}',
```
Comment (p0psicles) [inline medusa/metadata/mede8er.py]: Same

File: medusa/metadata/mede8er.py [hunk 3]
```diff
@@ -384,13 +380,13 @@ def write_ep_file(self, ep_obj):
 
         try:
             if not os.path.isdir(nfo_file_dir):
-                logger.log(u'Metadata directory did not exist, creating it at {path}'.format
-                           (path=nfo_file_dir), logger.DEBUG)
+                log.debug(u'Metadata directory did not exist, creating it at {location}',
+                          {'path': nfo_file_dir})
```
Comment (p0psicles) [inline medusa/metadata/mede8er.py]: Same

File: medusa/metadata/mede8er.py (hunk 2 above)
Comment (labrys) [inline medusa/metadata/mede8er.py]: good catch... thought i had fixed all of these

**Comment by labrys** [comment]
@p0psicles please update review

**Pull request merged**